packages/site/src/content/Content/index.tsx

```diff
@@ -9,30 +9,37 @@ export default {
   props: Props,
   mobileProps: MobileProps,
   component: {
-    element: `
-    return (
-    <Content spacing={"small"}>
-      <Card title="About me">
-        <Content spacing={"small"}>
-          <Heading level={2}>Sign up!</Heading>
-          <Text>
-            Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis
-            vestibulum. Nulla vitae elit libero, a pharetra augue. Nullam quis
-            risus eget urna mollis ornare vel eu leo. Vestibulum id ligula porta
-            felis euismod semper. Curabitur blandit tempus porttitor.
-          </Text>
-          <InputText placeholder="Name" />
-          <InputText placeholder="Phone" />
-          <InputText placeholder="Email" />
-          <InputText
-            multiline={true}
-            placeholder="Describe yourself"
-            name="describeAge"
-          />
+    element: `return (
```

Comment:
Considering this fits better in the iframe and still conveys what the component does, I'm good with this change.  I do think having the old example remain in storybook/become a future example in New Docs is a good idea so we can continue sharing patterns - but that's not something we should address here.